hoodie-common/src/main/java/com/uber/hoodie/common/HoodieCleanStat.java

```diff
@@ -81,6 +82,46 @@ public String getEarliestCommitToRetain() {
         return new Builder();
     }
 
+    public HoodieCleanStat addDeletePathPattern(String deletePathPattern) {
```

Comment:
This goes against the builder pattern followed to construct this immutable object. Can you do this right? Thanks.